config/locales/en/prior_authority.yml

```diff
@@ -48,16 +51,120 @@ en:
     assignments:
       create:
         no_unassigned_applications: There are no applications waiting to be allocated
+    service_types:
+      ae_consultant: A&E consultant
+      accident_reconstruction: Accident reconstruction
+      accountant: Accountant
+      anaesthetist: Anaesthetist
+      animal_behaviourist: Animal behaviourist
```

Comment:
i suppose there is no simple way we can avoid baking in knowledge of the individual service types. long term be good to have an API that both apps consume for this kind of look up data, including translations there i guess.

Reply:
I've added a ticket to Trello to discuss, cos you're right. I think in NSM they include translated values as part of the payload, but I think I remember David saying he regretted that approach.